tests/unit/asyn/wallet/test_main.py

```diff
@@ -139,6 +140,15 @@ def test_constructor(self):
 
         _test_wallet_values(self, wallet, "seed", "secp256k1")
 
+    def test_wallet_contructor_throws_on_empty_seed(self):
+        with self.assertRaises(XRPLAddressCodecException):
+            Wallet(
+                constants["regular_key_pair"]["secp256k1"]["public_key"],
+                constants["regular_key_pair"]["secp256k1"]["private_key"],
+                master_address=constants["regular_key_pair"]["master_address"],
+                seed="",
+            )
```

Comment:
This test case can be simplified I believe by removing the master_address piece and using a clearly invalid seed (as opposed to one which could be mistaken for an "empty" seed - aka not passing one in)

```suggestion
    def test_wallet_contructor_throws_with_invalid_seed(self):
        with self.assertRaises(XRPLAddressCodecException):
            Wallet(
                constants["regular_key_pair"]["secp256k1"]["public_key"],
                constants["regular_key_pair"]["secp256k1"]["private_key"],
                seed="abc",
            )
```

Reply:
Or add another test case for the empty string case as opposed to the "regular" error case